rust/src/ledger/account/mod.rs

```diff
@@ -81,6 +89,20 @@ pub struct ReceiptChainHash(pub String);
 pub struct Username(pub String);
 
 impl Account {
+    /// Time-locked balance (subtracted from circulating supply)
+    pub fn current_minimum_balance(&self, curr_global_slot: u32) -> u64 {
+        self.timing.as_ref().map_or(0, |t| {
+            if curr_global_slot < t.cliff_time {
+                t.initial_minimum_balance
+            } else {
+                t.initial_minimum_balance.saturating_sub(
+                    ((curr_global_slot - t.cliff_time) / t.vesting_period) as u64
+                        * t.vesting_increment,
```

Comment:
Why the multiplication by vesting increment?

Reply:
After the `cliff_time` (a slot number) has passed, every `vesting_period` slots, `vesting_increment` tokens are unlocked